package.json

```diff
@@ -62,11 +62,11 @@
     "lint-check": "tsc --noEmit && eslint \"*/**/*.{js,ts,tsx}\" NOTICE --max-warnings 0",
     "test": "run-s test:unit test:e2e",
     "test:unit": "jest --runInBand --config=./test/unit/jest.config.js",
-    "test:unit:coverage": "jest --runInBand --config=./test/unit/jest.config.js --coverage",
-    "test:unit:watch": "jest --runInBand --config=./test/unit/jest.config.js --coverage --watchAll",
+    "test:unit:coverage": "npm run test:unit -- --coverage",
+    "test:unit:watch": "npm run test:unit:coverage -- --watchAll",
```

Comment:
:warning: behaviour change
I don't use this command (nor know why we created it) so I won't be impacted.
If @aibcmars is ok with this change, let's do it (check the doc if there is any mention to it)

Reply:
Actually, the coverage was already run as part of the 'watch', so there is no behaviour change.